src/app/modules/home/components/login-form/login-form.component.html

```diff
@@ -0,0 +1,89 @@
+                            Sign In
+                        </button>
+
+
+                    </form>
+                </mat-card-content>
+            </mat-tab>
+
+            <mat-tab class="login__signup" label="Sign Up">
+                <mat-card-content class="signup__list">
+                    <form [formGroup]="registerForm" (ngSubmit)="register()">
+                        <mat-form-field class="signup__item">
+                            <input matInput formControlName="firstName" class="signup__firstname"
+                                placeholder="First Name" required name="firstName">
+                            <mat-icon matSuffix class="signup__firstname-icon icon">perm_identity</mat-icon>
+                            <mat-error *ngIf="getRegisterControls.firstName.invalid">First name is required</mat-error>
+                        </mat-form-field>
+
+                        <mat-form-field class="signup__item">
+                            <input matInput formControlName="lastName" class="signup__lastname" placeholder="Last Name"
+                                required name="lastName">
+                            <mat-icon matSuffix class="signup__lastname-icon icon">perm_identity</mat-icon>
+                            <mat-error *ngIf="getRegisterControls.lastName.invalid">Last name is required</mat-error>
+                        </mat-form-field>
+
+                        <mat-form-field class="signup__item">
+                            <input matInput formControlName="email" class="signup__email" placeholder="Email" required
+                                name="email">
+                            <mat-error *ngIf="getRegisterControls.email.invalid">Email is required (name@example.com)</mat-error>
+                            <mat-icon matSuffix class="signup__password-icon icon">email</mat-icon>
+                        </mat-form-field>
+
+                        <mat-form-field class="signup__item">
+                            <input matInput formControlName="password" class="signup__password" placeholder="Password"
+                                type="password" required minlength="6" name="password">
+                            <mat-error
+                                *ngIf="getRegisterControls.password.errors && getRegisterControls.password.errors.required">
+                                Password is required
+                            </mat-error>
+                            <mat-error
+                                *ngIf="getRegisterControls.password.errors && getRegisterControls.password.errors.minlength">
+                                Password must be at least 6 characters long with 1 symbol 
+                            </mat-error>
+                            <mat-icon matSuffix class="signup__password-icon icon">lock</mat-icon>
+                        </mat-form-field>
+                        <small class="signup__message">* - field is required</small>
```

Comment:
I'm not sure that is a good way to use 'small', may be better use style's to control a font size of element?